### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by rex4539** [comment]
@prokopyl Could you please review?

**Comment by rex4539** [comment]
Closing due to no interest.

File: docs/make_chapters.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def __str__(self):
 class CommentLine(Line):
     """
     A line that was a comment in the source document.
-    It will be outputed as normal text.
+    It will be outputted as normal text.
```
Comment (PieterPenninckx) [inline docs/make_chapters.py:23]: I think this should in fact be "output" (irregular verb)?

File: docs/make_chapters.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ class CommentLine(Line):
 class CodeLine(Line):
     """
     A line that was source code in the source document.
-    It will be outputed as quoted code.
+    It will be outputted as quoted code.
```
Comment (PieterPenninckx) [inline docs/make_chapters.py:32]: Same remark.

**Review by PieterPenninckx — COMMENTED** [review]
Maybe some grammar errors aren't yet corrected (I'm not sure: I'm not an English speaker myself), but it's definitely an improvement.

**Comment by PieterPenninckx** [comment]
Hi @rex4539 I think this PR didn't get a review due to lack of time rather than lack of interest. Looks good to me.

**Pull request closed without merging**